### PR TITLE
Fix filter stake less then

### PR DESCRIPTION
**Pull request by baidang201**

### CODE REVIEW

File: backend/client.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function testWorkerRequest() {
         page: 1,
         pageSize: 5,
         filterRuning: false,
-        filterStakeEnough: false,
+        filterStakeLessThen: true,
```
Comment (jasl) [inline backend/client.js]: `then` -> `than`?

File: backend/client.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ function testWorkerRequest() {
...
         filterCommissionLessThen: true,
```
Comment (jasl) [inline backend/client.js]: same above?

**Comment by amphineko** [comment]
No enough data to test.

UI can now correctly retrieves miners with not enough stakes. But no miners with enough stakes to test if they could be filtered out.

**Review by amphineko — COMMENTED** [review]
No enough data to test.

UI can now correctly retrieves miners with not enough stakes. But no miners with enough stakes to test if they could be filtered out.

File: backend/client.js (hunk 2 above)
Comment (baidang201) [reply backend/client.js]: fix

File: backend/client.js (hunk 1 above)
Comment (baidang201) [reply backend/client.js]: fix

**Pull request merged**